### DISCUSSION TITLE
Pratik Resume

### DISCUSSION THREAD

**Discussion by pratiks05** [recodehive/Stackoverflow-Analysis#350] [Q&A] [answered]
[Professional Resume-Pratik Singh.pdf](https://github.com/user-attachments/files/16324454/Professional.Resume-Pratik.Singh.pdf)
I am trying to apply for web developer role. Can you please help me out like what changes should I make in my resume?

**Comment by sanjay-kv** [comment]
[pratik.pdf](https://github.com/user-attachments/files/16342907/pratik.pdf)
[ATS Version of CV (10).docx](https://github.com/user-attachments/files/16342908/ATS.Version.of.CV.10.docx)

**Reply by pratiks05** [reply]
[Pratik Kumar Singh.pdf](https://github.com/user-attachments/files/16649817/Pratik.Kumar.Singh.pdf)

Can you please provide a review for the resume.

**Reply by sanjay-kv** [reply]
yes, it will take a day or two time

**Reply by pratiks05** [reply]
Can you please provide the review or ATS score of the updated resume given above.

**Comment by Jisha-tr** [comment, marked as answer]
The resume seems good, i think new template would be great.

**Comment by 10senura** [comment]
The resume seems good, i think new template would be great.

**Comment by 10senura** [comment]
greate

**Comment by Utkarshell** [comment]
bro its awesome

**Comment by BernaolaBarrientos** [comment]
Good resume — it's clear and well-organized.

**Comment by HARIHARANS24** [comment]
The resume seems good, i think new template would be great.

**Comment by sandun-harshana** [comment]
i think new template would be great

**Comment by sandun-harshana** [comment]
it's clear and well-organized

**Comment by umidjon0211** [comment]
The resume seems good, i think new template would be great.

**Reply by umidjonnajot5910** [reply]
good

**Comment by tawfique02** [comment]
The resume looks quite solid! A fresh new template could make it even more impressive and stand out better. Great work overall! 👏
